### ISSUE TITLE
Real aircraft feedback

### ISSUE THREAD

**Issue by algefaen** [c172p-team/c172p-detailed#897]
Hi! This is not a bug report, more a place to gather feedback from real c172 flying.

Last summer I said I would get access to our club's c172p and do some investigation for this project. Unfortunately Shit Happened[tm] and I didn't get my PPL done before the club had already sold their c172p. We still have a c172n and a c172sp G1000 though - both of which I intend to fly a lot this spring.

I had booked the c172n for some circuit practice today, but we got something like 30cm of snow, so that didn't happen. Instead I decided to grab some videos for you guys, and get familiar with the aircraft. Sorry about the lighting (closed hangar), potato focus (I blame Samsung). It's not a c172p but the c172n is pretty damn close, so I hope most of it is directly usable.

Gallery: https://goo.gl/photos/pqfDvnnNjDW7Vff29

If someone has any questions that might be answered from looking at an aircraft, measuring stuff, trying stuff out in the air - anything like that - ask me anything! And other pilots with access to real aircraft are also welcome to help out. (=

(One caveat: It's a typical club aircraft, and I'm an amateur pilot. No bitching about the state of the aircraft or my handling of it, please.) (=

**Comment by wlbragg** [comment]
Really nice, good stuff, thank you!

**Comment by tigert** [comment]
Yeah,  This is exactly why it's so awesome to have an open source flight simulator project. Because of folks like us. Different backgrounds and ideas working together. Happy to have you around :-) Welcome!

The OH-CTL livery and some interior stuff has its reference on our club's Cessna 172M. Some pics of the real thing: http://mik.1g.fi/kuvat/Kerhon+kalusto/OH-CTL/ - and the float version is not fantasy either - OH-CTL got the 180HP "Penn Yann Aero Superhawk" upgrade and was later sold to a newly established seaplane club where it was put on floats.. It had the Penn Yann Superhawk 180HP conversion done to it - looks like yours has the same sticker in the prop?
